linkerd/protocol/h2/src/main/scala/io/buoyant/linkerd/protocol/h2/H2AccessLogger.scala

```diff
@@ -0,0 +1,83 @@
+package io.buoyant.linkerd.protocol.h2
+
+import com.twitter.finagle.{Service, ServiceFactory, SimpleFilter, Stack, Stackable}
+import com.twitter.finagle.buoyant.h2.{Request, Response}
+import com.twitter.logging._
+import com.twitter.util.{Time, TimeFormat}
+
+case class H2AccessLogger(log: Logger) extends SimpleFilter[Request, Response] {
+
+  def apply(req: Request, svc: Service[Request, Response]) = {
+    val reqHeaders = req.headers
+    val remoteHost = "-"
+    val identd = "-"
+    val user = "-"
+    val referer = "-"
```

Comment:
I believe the `referer` header field may be present on h2 requests, so this probably ought to be 
```scala
reqHeaders.get("referer").getOrElse("-")
```

Reply:
happy to change it to this. kevin and i  did loop over all headers and referrer was not a key in that sequence: here's the output for you review, in case you see the referer being associated to a different key?

```
:method
POST
:scheme
http
:path
/helloworld.Hello/Greeting
:authority
localhost:4142
content-type
application/grpc
user-agent
grpc-go/1.8.0
te
trailers
l5d-dst-service
/svc/localhost:4142
via
h2 linkerd
l5d-dst-client
/$/inet/127.1/8888
l5d-ctx-trace
BFa50jQ7Na/DuK+B+vMbkMO4r4H68xuQAAAAAAAAAAA=
l5d-reqid
c3b8af81faf31b90
```

the current output after changing the value of `referer` to `reqHeaders.get("referer").getOrElse("-")` is still `"-"`

Reply:
As we discussed, i think that this header is not always set, but can be present depending on the request --- I think we should attempt to log it if it is present.